src/firefly/js/tables/TableUtil.js

```diff
@@ -486,6 +486,29 @@ export function getRowValues(tableModel, rowIdx) {
     return get(tableModel, ['tableData', 'data', rowIdx], []);
 }
 
+/**
+ * Shape values according to arraySize specification.
+ * @param {TableColumn} col
+ * @param {Object} val
+ * @returns {*} shaped value, if a column is an array or an unchanged value
+ */
+export function convertToArraySize(col, val) {
+
+    const aryDim = (col.arraySize || '').split('x');
+
+    if (col.type === 'char' && aryDim.length > 0) {
+        aryDim.shift();    // remove first dimension because char array is presented as string
+    }
+
+    if (aryDim.length > 1) {
+        for(let i = 0; i < aryDim.length-1; i++) {
+            val = chunk(val, parseInt(aryDim[i]));
```

Comment:
`parseInt` is quirky, meaning you may get unexpected results, e.g. `1x2` -> 1, `1.23` -> 1.
If you're ok with that, then it's fine.  Otherwise, we have a WebUtil.strictParseInt you can use instead.